internal/db/nosql/cassandra/cassandra.go

```diff
@@ -136,7 +137,12 @@ func New(opts ...Option) (Cassandra, error) {
 	c := new(client)
 	for _, opt := range append(defaultOpts, opts...) {
 		if err := opt(c); err != nil {
-			return nil, errors.ErrOptionFailed(err, reflect.ValueOf(opt))
+			err = errors.ErrOptionFailed(err, reflect.ValueOf(opt))
```

Comment:
I think we should check the error type.
the following is example.
https://github.com/vdaas/vald/projects/3#card-44665812